.github/workflows/make-branch.yml

```diff
@@ -0,0 +1,31 @@
+name: Create branch
+on:
+  # to be run once every 3 weeks for a 7.yy.0 release, so we can get a stable 7.yy.x branch for use by downstream consumers
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'The branch to create. Should be in format 7.yy.x'
+        required: true
+      branchfrom:
+        description: 'The source branch from which to branch, eg., main'
+        default: 'main'
+      forceflag:
+        description: 'To force creation of .x branch, use --force flag here'
+        default: ''
+jobs:
+  build:
+    name: Create branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+      - name: Create branch
+        run: |
+          git config --global user.name "Mykhailo Kuznietsov"
+          git config --global user.email "mkuznets@redhat.com"
+          export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
```

Comment:
should this be CHE_INCUBATOR_BOT_TOKEN?

Reply:
removed it for now to try with default token